k8s/spack/spackbot-spack-io/secrets-dummy.yaml

```
@@ -0,0 +1,20 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: spack-bot-credentials
+  namespace: spack
```

Make sure to include the following annotation.  This is needed so that flux doesn't confuse this for a real secret!

```suggestion
  namespace: spack
  annotations:
    fluxcd.io/ignore: "true"
```

---

done